SocialCareCaseViewerApi/V1/Controllers/FormSubmissionController.cs

```diff
@@ -40,6 +42,20 @@ public IActionResult GetSubmissionById(string submissionId)
             return Ok(submission);
         }
 
+        /// <summary>
+        /// Lists all in-progress case submissions
+        /// </summary>
+        /// <response code="200">Success. Returns a list of any in progress case submissions</response>
+        [ProducesResponseType(typeof(List<CaseSubmissionResponse>), StatusCodes.Status200OK)]
+        [HttpGet]
+        [Route("status/in-progress")]
```

Comment:
probably this can just be `/submissions` to stick to REST conventions - no need for the extra path bits

at most we might want to use a query string parameter like `?status=` to filter more intelligently down the line, but not needed for now

Reply:
👍🏾 I can update this